spk/maraschino-custom/Makefile

```diff
@@ -0,0 +1,49 @@
+SPK_NAME = maraschino-custom
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 1
+SPK_ICON = src/maraschino-custom.png
+DSM_UI_DIR = app
+
+DEPENDS  =
+SPK_DEPENDS = "python>2.7.3-3:git"
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Maraschino Custom allows you to run the fork of your choice of Maraschino. You will be asked for the fork\'s git URL during install.
+DESCRIPTION_FRE = Maraschino Custom vous permet d\\\'ex�cuter le fork de votre choix de Maraschino. Vous aurez besoin de saisir l\\\'URL git du fork durant l\\\'installation.
+ADMIN_PORT = 8262
+RELOAD_UI = yes
+DISPLAY_NAME = Maraschino-custom
```

Comment:
To keep in line with the other Custom packages, change this to `Maraschino Custom`